AppCenter/AppCenter/Internals/Sender/MSOneCollectorIngestion.h

```diff
@@ -0,0 +1,24 @@
+#import "MSHttpSender.h"
+
+extern NSString *const kMSOneCollectorApiVersion;
+extern NSString *const kMSOneCollectorApiPath;
+extern NSString *const kMSOneCollectorContentType;
+extern NSString *const kMSOneCollectorApiKey;
+extern NSString *const kMSOneCollectorClientVersionKey;
+
+// Assign value to avoid "format is not a string literal" warning.
+static NSString *const kMSOneCollectorClientVersionFormat = @"ACS-iOS-ObjectiveC-no-%@-no";
```

Comment:
Can you add a comment "<sdktype>-<platform>-<language>-<projection>-<version>-<tag>" for kMSOneCollectorClientVersionFormat?